frontend/sw360-portlet/src/main/webapp/html/components/editRelease.jsp

```diff
@@ -241,9 +241,9 @@
             var beforeComma = vendorInfo.substr(0, vendorInfo.indexOf(","));
             var afterComma = vendorInfo.substr(vendorInfo.indexOf(",") + 1);
 
-            $('#<%=Release._Fields.VENDOR_ID.toString()%>').val(beforeComma.trim());
-            $('#<%=Release._Fields.VENDOR_ID.toString()%>Display').val(afterComma.trim());
-        }
+             $('#<%=Release._Fields.VENDOR_ID.toString()%>').val(beforeComma.trim());
```

Comment:
@ChristophNiehoff, I don't think you've added these spaces with a particular purpose in mind, have you? Could you, maybe, remove them real quick so that I coulc merge this PR?